src/db/models/ledger.ts

```diff
@@ -0,0 +1,27 @@
+import { DataTypes, Model } from 'sequelize';
+import sequelize from '../db';
+
+export default class Ledger extends Model {
+  public id!: number;
+  public userid!: number;
+}
+
+Ledger.init({
+  id: {
+    type: DataTypes.INTEGER.UNSIGNED,
+    autoIncrement: true,
+    primaryKey: true,
+  },
+  userid: {
+    type: DataTypes.INTEGER.UNSIGNED,
+  },
+}, {
+  sequelize,
+  tableName: 'ledgers',
+});
+
+Ledger.hasOne(User, {
```

Comment:
where is user?

Reply:
without user model, app will crash I assume

Reply:
it is better to get payee and account first, ledger user after.

because payee and account has relation to ledger